site/content/docs/main/api-types/restore.md

```diff
@@ -126,6 +128,26 @@ spec:
             - /bin/ash
             - -c
             - echo -n "DEADFEED" >> /restores/pvc2-vm/deadfeed
+      - exec:
+          # The container name where the hook will be executed. Defaults to the first container.
```

Comment:
If this defaults to the first container, can it be marked as `Optional`? This question also applies to the other fields below which also have default values.

Reply:
Added to all.